src/routing_table.c

```diff
@@ -371,6 +371,15 @@ olsr_delete_rt_path(struct rt_path *rtp)
     current_inetgw = NULL;
   }
 
+  /* remove reference from rt_best */
+  struct rt_entry* rt;
```

Comment:
this will cause compilation failures.
C99 does not allow this. Put declarations at the beginning of the function or at the beginning of a block.
did you compile at all?

Reply:
definitely, but you're right anyway, I missed the ISO C90 warning. Didn't run with warnings-as-error flag just now.